_config/site.yml

```diff
@@ -1,201 +0,0 @@
-    symlinker:
-      from: ../stylesheets
-      to: ./_site/stylesheets-source
-    asciidoctor:
-      :attributes:
-        base_url: http://localhost:4242
-        imagesdir: /images
-    integrations:
-      java:
-        name: Java
-      connect:
-        name: Kafka Connect
-      mysql:
-        name: MySQL
-      mongodb:
-        name: MongoDB
-      postgresql:
-        name: PostgreSQL
-      oracle:
-        name: Oracle
-      sqlserver:
-        name: SQL Server
-      cassandra:
-        name: Cassandra
-      db2:
-        name: Db2
-      vitess:
-        name: Vitess
-  staging:
-    jborg_fonts_url: https://static.jboss.org/theme/fonts
-    jborg_images_url: https://static.jboss.org/theme/images
-    jborg_js_url: https://static.jboss.org/theme/js
-    project_images_url: https://static.jboss.org/example/images
-    bootstrap_css_url: https://static.jboss.org/theme/css/bootstrap-community/3.2.0.2/bootstrap-community
-    bootstrap_js_url: https://static.jboss.org/theme/js/libs/bootstrap-community/3.2.0.2/bootstrap-community
-    base_url: https://debezium.io/staging
-    disqus_developer: true
-    wget:
-      enabled: false
-    asciidoctor:
-      :attributes:
-        base_url: https://debezium.io/staging
-        imagesdir: /staging/images
-    integrations:
-      java:
-        name: Java
-      connect:
-        name: Kafka Connect
-      mysql:
-        name: MySQL
-      mongodb:
-        name: MongoDB
-      postgresql:
-        name: PostgreSQL
-      oracle:
-        name: Oracle
-      sqlserver:
-        name: SQL Server
-      cassandra:
-        name: Cassandra
-      db2:
-        name: Db2
-      vitess:
-        name: Vitess
-  production:
-    minified: .min
-    css_minifier: enabled
-    js_minifier: enabled
-    html_minifier: enabled
-    jborg_fonts_url: https://static.jboss.org/theme/fonts
-    jborg_images_url: https://static.jboss.org/theme/images
-    jborg_js_url: https://static.jboss.org/theme/js
-    project_images_url: https://static.jboss.org/example/images
-    bootstrap_css_url: https://static.jboss.org/theme/css/bootstrap-community/3.2.0.2/bootstrap-community
-    bootstrap_js_url: https://static.jboss.org/theme/js/libs/bootstrap-community/3.2.0.2/bootstrap-community
-    base_url: https://debezium.io
-    wget:
-      enabled: false
-    asciidoctor:
-      :attributes:
-        base_url: https://debezium.io
-        imagesdir: /images
-    integrations:
-      java:
-        name: Java
-      connect:
-        name: Kafka Connect
-      mysql:
-        name: MySQL
-      mongodb:
-        name: MongoDB
-      postgresql:
-        name: PostgreSQL
-      oracle:
-        name: Oracle
-      sqlserver:
-        name: SQL Server
-      cassandra:
-        name: Cassandra
-      db2:
-        name: Db2
-      vitess:
-        name: Vitess
-    deploy:
-      host: github_pages
-      branch: master
-
-google_analytics:
-  account: UA-76464546-1
-  anonymizeIp: true
```

Comment:
Can we set these attributes for the Jekyll GA integration, too. If not supported by the plug-in, take a look at the rendered output on the current website and add the attributes straight to the template.